app/src/main/kotlin/com/mapzen/erasermap/view/RouteModeView.kt

```diff
@@ -50,9 +50,6 @@ class RouteModeView : LinearLayout, RouteViewController, ViewPager.OnPageChangeL
     val mapListToggle: MapListToggleButton by lazy {
         findViewById(R.id.map_list_toggle)as MapListToggleButton
     }
-    val routeCancelButton: ImageView by lazy {
```

Comment:
@souperneon removing it from both places was what we had discussed and what is in your mocks. let me show you what this looks like on the device to hopefully remove confusion though!